protostar/commands/cairo1_commands/build_cairo1_command.py

```diff
@@ -67,10 +67,15 @@ async def _build_contract(
             f"Multiple files found for contract {contract_name}, "
             f"only one file per contract is supported in cairo1!"
         )
+        cairo1.compile_starknet_contract_to_sierra_from_path(
+            input_path=contract_paths[0],
+            cairo_path=cairo_path,
+            output_path=output_dir / (contract_name + ".sierra.json"),
+        )
         cairo1.compile_starknet_contract_to_casm_from_path(
             input_path=contract_paths[0],
             cairo_path=cairo_path,
-            output_path=output_dir / (contract_name + ".json"),
+            output_path=output_dir / (contract_name + ".casm.json"),
```

Comment:
Can we not use the sierra as an input here? Seems like a waste of time to compile all the way from cairo

Reply:
There is no `starknet_sierra_to_casm` function in `bindings`

Reply:
As the whole compilation happens quickly enough, we have decided that we can leave it as is for now (discussed in slack)

Reply:
I don't think there was an agreement that the compilation is quick enough. From what I understand what happened was: We agreed that if we don't need an additional method, we shouldn't add it. Then you said that the compilation is fast so we don't need it. Looking at this code, I'd reconsider adding this shortcut method `sierra -> casm`. What happens here now is we do the same thing (compile `cairo -> sierra`) twice for no good reason while we could do this just once. I don't see why we should do so.

Reply:
I created a cairo PR for this https://github.com/software-mansion-labs/cairo/pull/61